src/components/Search/SearchBox.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import React from 'react';
+import { Search } from 'lucide-react';
+interface SearchBoxProps {
+  search: string;
+  setSearch: React.Dispatch<React.SetStateAction<string>>
+}
+
+export default function SearchBox({ search , setSearch}: SearchBoxProps) {
+  return (
+    <div>
+      <div className="mb-4">
+        <div className="flex items-center bg-gray-100 rounded px-2">
+          <Search className="text-gray-400 mr-2" />
+          <input
+            type="text"
+            value={search}
+            onChange={(e) => setSearch(e.target.value)}
+            placeholder="Search commands..."
+            className="bg-transparent border-none outline-none w-full py-2 text-gray-900 placeholder-gray-500"
+          />
+        </div>
+      </div>
+      <div className="space-y-2">
```

Comment:
Initially we don't need to set values, it can be empty.
<img width="845" alt="image" src="https://github.com/user-attachments/assets/2c12096b-79b8-415c-9968-768c5921358c">